main_menu.py

```diff
@@ -0,0 +1,149 @@
+import curses
+
+
+def read_input(window):
+    return window.getch()
+
+
+def save_file():
+    return True
+
+
+class MainMenu:
+    def __init__(self, screen):
+        self.screen = screen
+        self.pressed_key = ord('z')
+        self.top = "--- Tower Explorer ---"
+        self.logo = ["     |>>>  ", "     |     ", " _  _|_  _ ", "|;|_|;|_|;|",
+                     "\\\.    .  /", " \\\:  .  / ", "  ||:   |  ", "  ||:.  |  ",
+                     "  ||:  .|  ", "  ||:   |  ", "  ||: , |  ", "  ||:   |  ",
+                     "  ||:   |  ", "  ||: . |  ", "  ||_   |  ", " ", " "]
+
+        if save_file():
+            self.menu_items = ["[n] Neues Spiel",
+                               "[f] Fortsetzen", "[b] Beenden"]
+        else:
+            self.menu_items = ["[n] Neues Spiel", "[b] Beenden"]
+        self.credits = "[c] Credits"
+
+    def print(self):
+        # clear current screen
+        self.screen.clear()
+        # get a Tupel (y, x) - height, width of the window
+        size = self.screen.getmaxyx()
+
+        # create new window for menu
+        menu_item_win = curses.newwin(size[0], size[1], 0, 0)
+        # y_pos_offset to set items vertical below each other
+        y_pos_offset = int(size[0] / 7) - 2
```

Comment:
in python kann man integer division mit `//` machen. Also `size[0] // 7` statt `int(...)`, fände das ganz gut wenn wir das verwenden